### PR TITLE
lvl-max

### PR DESCRIPTION
**Pull request by KimuraYu45z**
I created `lvl-max` which is a template uses all modules in `cosmos-sdk`.
In concrete, `crisis`, `mint`, `gov`, `evidence`, `upgrade`.
The order of modules in code is referred to the codes of `gaia`.

I already checked that the code created by `lvl-max` scaffold can be compiled.

I think that the name `lvl-max` should be changed properly, but I would appreciate if this PR is accepted because I just want fully module using template.

Thanks
Kimura Yu (LCNEM, Inc. Japanese leading cosmos community company)

### CODE REVIEW

File: templates/lvl-max/app/sim_test.go.tmpl [hunk 1]
```diff
@@ -0,0 +1,296 @@
+package app
+
```
Comment (marbar3778) [inline templates/lvl-max/app/sim_test.go.tmpl]: not sure if we need this file as it has links to the cosmos-sdk in particular

Comment (KimuraYu45z) [reply templates/lvl-max/app/sim_test.go.tmpl]: I see.
I often want this file in the flow like below:

- scaffolding chain
- scaffolding original module
- chain uses the module
- testing the chain

copy-and-paste-ing integration test of module everytime is bothersome.

It is my opinion but I'll follow your final dicision.

Comment (marbar3778) [reply templates/lvl-max/app/sim_test.go.tmpl]: okay we can keep it but `simapp.AppStateFn(app.Codec(), app.SimulationManager())` is using the default genesis of simapp i believe so if you add new modules you will have to add in these files manually. We can merge it as is now, and I will do a longer simulation follow up to app-lvl1 and this one

**Comment by marbar3778** [comment]
could you add this to the app scaffold docs, please then ill merge

File: templates/lvl-max/app/sim_test.go.tmpl (hunk 1 above)
Comment (KimuraYu45z) [reply templates/lvl-max/app/sim_test.go.tmpl]: Okay, thank you very much.

**Comment by KimuraYu45z** [comment]
Ok.

**Comment by KimuraYu45z** [comment]
Done.

**Comment by marbar3778** [comment]
thank you 🙏 😊

**Pull request merged**